datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/security/SecurityConfig.java

```diff
@@ -61,7 +61,7 @@ public SecurityFilterChain filterChain(HttpSecurity http, MvcRequestMatcher.Buil
                         .requestMatchers(SWAGGER_RESOURCES).permitAll()
                         .requestMatchers(mvc.pattern("/static/**")).permitAll()
                         .requestMatchers(mvc.pattern(SecurityConstants.AUTH_LOGIN_URL)).permitAll()
-                        .requestMatchers(mvc.pattern(HttpMethod.OPTIONS, "/**")).permitAll()
+                        .requestMatchers(mvc.pattern(HttpMethod.GET, "/**")).permitAll()
```

Comment:
Make sure that frontend works! I think Options is needed for cors

Reply:
Checked. Frontend also works after this update. I created another branch and I made the same update. everything is working. But now i must to close this PR and pull another one because there is probably some file missing in the frontend part.

Reply:
you can rebase your branch on top of add_ECDH